packages/boxel-ui/test-app/app/templates/index.hbs

```diff
@@ -1,4 +1,5 @@
 {{page-title 'Boxel Components'}}
+<BasicDropdownWormhole />
```

Comment:
According to the [migration documentation](https://ember-power-select.com/docs/migrate-7-0-to-8-0) this should also happen in `host` but doing so resulted in strange Percy diffs like this:

<img width="947" alt="Build #3755 - Cardstack - Percy | Visual testing as a service 2024-03-14 12-35-54" src="https://github.com/cardstack/boxel/assets/43280/de7004ee-306b-40ba-b212-fef33e93a97f">

When I used the application I didn’t see anything like this but since the existing setup worked fine with dropdowns, I left it.

Reply:
So we r not following the recommendation of the migration docs?

Reply:
@backspace try moving the <BasicDropdownWormhole/> after <CardPrerender/>. I think that works

Reply:
When I put it there I still get the strange [Percy diffs](https://percy.io/Cardstack-1/-cardstack-host/builds/33245096?utm_campaign=Cardstack-1&utm_content=-cardstack-host&utm_source=github_status_private), also when I [put it after `CardPrerender`](https://percy.io/Cardstack-1/-cardstack-host/builds/33245572?utm_campaign=Cardstack-1&utm_content=-cardstack-host&utm_source=github_status_private). I don’t love deviating from the documentation but I think `<BasicDropdownWormhole />` is just emitting that container so it’s probably fine?